content/embeds/tryout-redisearch-2.md

```diff
@@ -0,0 +1,93 @@
+## Creating indexes
+
+Let's create a new index called "myIdx". When you define the index, you
+must pass in the structure of the data you are adding to the index.
+In this example, we have four things we input: title, body, url, and
+value. In this example, we have three TEXT and one NUMERIC values. The
+title has a weight of 5.0.
+
+1. Connect to redis.
+
+    ```sh
+    $ redis-cli -p 12543
+    127.0.0.1:12543>
+    ```
+
+1. Create the schema with the command:
+
+    ```sh
+    127.0.0.1:12543> FT.CREATE myIdx PREFIX 1 "doc:" SCORE_FIELD "value" SCHEMA title TEXT body TEXT url TEXT value NUMERIC
+    ```
+
+    This command indexes the hashes that have the prefix "doc".
+    By default, all hashes get a score of 1.0 but here the index uses the `value` field as the score for this hash.
```

Comment:
The meaning of this sentence isn't clear. Please break into two sentence, and explain that the SCORE_FIELD directive specified a value whose value can override the default score for a document.